CODEOWNERS

```diff
@@ -6,3 +6,4 @@
 **/alchemical_model     @abmazitov
 **/pet                  @spozdn @abmazitov
```

Comment:
@spozdn could you accept the invitation to the organization? Otherwise this the CODEOWNER integration will not work. Unless you prefer for @abmazitov to handle the metatrain side of things by himself?

Reply:
Solved in https://github.com/metatensor/metatrain/pull/400